sass/components/_feature-media.scss

```diff
@@ -0,0 +1,34 @@
+%feature-media {
+  h2 {
+    font-family: $secondary-font;
+    font-weight: $font-weight-sb;
+    font-size: $font-regular;
+    text-transform: uppercase;
+  }
+  p {
+    font-family: $primary-font;
+    font-weight: $font-weight-l;
+    font-size: $font-smaller;
+    line-height: $primary-lh;
+  }
+}
+
+.why-1 {
+  @extend %feature-media;
+  li:hover {
+    [class^=icon] {
+      color: border(primary);
+      border-color: color(accent);
+    }
+  }
+}
+
+.why-2 {
+  @extend %feature-media;
+  li:hover {
+    [class^=icon] {
+      color: border(primary);
+      border-color: color(accent);
+    }
```

Comment:
Looks like this block duplicates `.why-1`? No real difference between `.why-1` and `.why-2`?